internal/test/wmcb/wmcb_test.go

```diff
@@ -63,6 +75,7 @@ var (
 		Value:  "Windows",
 		Effect: v1.TaintEffectNoSchedule,
 	}
+	dockerRuntime = flag.Bool("dockerRuntime", true, "Container runtime to be used")
```

Comment:
where is flag.Parse() being called?
this flag should be defined where the other flags are being defined, flags come command line arguments and should be processed at the start of the program, not throughout the code.